File: Easy_python/1134_Armstrong_number.py
class Solution:
    def isArmstrong(self, N: int) -> bool:
        length=len(str(N))
        eachpos=list(str(N))
        listsum=0
        for i in range(len(eachpos)):
            listsum=listsum+pow(ord(eachpos[i])-ord('0'),length)
        if listsum==N:
            return True
        else:
            return False

File: Easy/1134_Armstrong_number.cpp
class Solution {
public:
    bool isArmstrong(int N) {
        vector<int> arr;
        int number=N;
        int count=0;
        while(number/10!=0)
        {
            arr.push_back(number%10);
            number=number/10;
            count+=1;
        }
        arr.push_back(number%10);
        count+=1;
        int sum=0;
        for(int i=0; i < arr.size();i++)
        {
            sum+=pow(arr[i],count);
        }
        if(sum == N)
            return true;
        else
            return false;
    }
};
